src/ResourceManager/Network/Commands.Network/LocalNetworkGateway/NewAzureLocalNetworkGatewayCommand.cs

```diff
@@ -66,7 +66,7 @@ public class NewAzureLocalNetworkGatewayCommand : LocalNetworkGatewayBaseCmdlet
             Mandatory = false,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "The local network gateway's ASN")]
-        public uint Asn { get; set; }
```

Comment:
I am not sure why any of these changes are necessary.  As far as I can etell, none of these property types have changed in the underlying types, so any mapping issues would also apply to the previous version (10.1.2) of the management library.  You can see the source code for the previous management library here: https://github.com/DeepakRajendranMsft/azure-sdk-for-net/blob/1c39c6683ff610bd4dc4f31a3a18b00e0fe71976/src/SDKs/Network/Management.Network/Generated